firstvoices/backend/serializers/validators.py

```diff
@@ -23,6 +23,23 @@ def __call__(self, value, serializer):
             raise serializers.ValidationError(self.message)
 
 
+class SameVisibilityAsSite:
```

Comment:
Nitpick: since this checks >= rather than ==, maybe we should call it something like MeetsSiteVisibility rather than "same"?